Include/Acidanthera/Protocol/OcLog.h

```diff
@@ -137,6 +138,7 @@ struct OC_LOG_PROTOCOL_ {
   UINTN                  DisplayLevel;  ///< The error level visible onscreen.
   UINTN                  HaltLevel;     ///< The error level causing CPU dead loop.
   EFI_FILE_PROTOCOL      *FileSystem;   ///< Log file system root, not owned.
+  EFI_FILE_PROTOCOL      *LogFile;      ///< Log file, owned. Unsafe logging only.
```

Comment:
Please move it all the way down, this is unsafe. In the Revision-model, instances of lesser revision must be prefixes of those of a higher revision. I.e., you can only append (or use previously reserved data), to preserve backwards-compatibility. This is of course not critical here, but I also see no good reason to not do it.